components/style/themes/default.less

```diff
@@ -108,8 +108,9 @@
 @btn-default-bg         : #fff;
 @btn-default-border     : @border-color-base;
 
-@btn-danger-color      : #fff;
-@btn-danger-bg         : @red-6;
+@btn-danger-color      : @red-6;
```

Comment:
I've a suggestion here (Which I would've raised as a separate PR if I hadn't seen this one).
Can you this variable to `@error-color`? 
https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less#L17

That way you only need to override error-color to get the desired color on the danger button. 
Makes sense? @afc163 @benjycui